src/apps/VirtualApp/VirtualClient.js

```diff
@@ -526,6 +526,7 @@ class VirtualClient extends Component {
       },
       error: (error) => {
         Janus.log('Error attaching plugin: ' + error);
+        captureException(error, {source: 'Videoroom'});
```

Comment:
This are all janus error callbacks. Should use captureMessage, not exception.

Reply:
Fixed.